src/Common/tests/System/Net/Configuration.Certificates.cs

```diff
@@ -18,9 +18,23 @@ public static partial class Certificates
         {
             private const string CertificatePassword = "testcertificate";
             private const string TestDataFolder = "TestData";
-            private static Mutex m = new Mutex(false, "Global\\CoreFXTest.Configuration.Certificates.LoadPfxCertificate");
+
+            private static Mutex m;
             private const int MutexTimeout = 120 * 1000;
 
+            static Certificates()
+            {
+                if (PlatformDetection.IsUap)
```

Comment:
https://github.com/dotnet/coreclr/issues/11306 is talking about porting the mutex fix to CoreCLR as it was fixed in CoreRT: https://github.com/dotnet/corert/commit/78fae17f5434839a8720603973ff53f27e27f916

So that is why I thought UapAot is not hitting that issue. 
cc: @kouvel